.github/workflows/ci.yml

```diff
@@ -51,10 +51,8 @@ jobs:
           python -m pip install --upgrade pip setuptools wheel
           python -m pip install --upgrade --upgrade-strategy eager -r requirements/dev.txt
           python -m pip install --upgrade --upgrade-strategy eager -e .
-          if [ "$RUNNER_OS" != "Windows" ] && ${{ matrix.python-version != '3.10.6' }}; then  # Skip Prophet tests on Windows as installation complex. Skip on Python 3.10 as not supported.
-            python -m pip install --upgrade --upgrade-strategy eager -e .[prophet]
-          fi
-          if [ "$RUNNER_OS" == "Linux" ]; then  # Currently, we only support KeOps on Linux.
+          python -m pip install --upgrade --upgrade-strategy eager -e .[prophet]
+          if [ "$RUNNER_OS" == "Linux"]; then  # Currently, we only support KeOps on Linux.
```

Comment:
The test coverage has dropped because the `keops` CI is not running. This is because a space is missing from here:

```yaml
if [ "$RUNNER_OS" == "Linux"]
```

Reply:
Good spot, will change it back. Weitd think is I don't think I edited that line, my editor must've done it automatically which is not great...

Reply:
Yeh I only know about this issue as ran into the same thing previously. Now I'm wondering if the editor did it last time too...